qa/TL0_python-self-test_xavier/test_body.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+
+test_nose() {
+    # we are not able to easily install this packages in xavier for aarch64 so filter it out
+    # also there is no nvJPEG on xavier so don't run any test with the ImageDecoder having
+    # the device explicitly set
+    EXCLUDE_PACKAGES=(
+        "cv2"
+        "scipy"
+        "librosa"
+        "image[_]*decoder\(device\s*="
```

Comment:
Anything worth testing stays after that? :P

Did you try swapping the image_decoder the way we discussed?

Reply:
It is too hacky and it just doesn't work, there is too many different cases.
Not much but still some basic tests can be executed.